bot/converters.py

```diff
@@ -15,6 +15,25 @@
 log = logging.getLogger(__name__)
 
 
+def string(*values, preserve_case: bool = False) -> t.Callable[[str], str]:
```

Comment:
Not sure on this being lowercase and not very descriptive of it's purpose.

Maybe something like `AllowedStrings` or `AcceptedStrings` instead?

Reply:
I felt it may be more confusing to name it as a class since passing parameters to converters is uncharacteristic.

Yes, I can make the name more descriptive.